HLTrigger/HLTfilters/test/triggerResultsFilter_by_PathStatus.py

```diff
@@ -24,17 +32,9 @@
     lvl1DefaultLabel = cms.string('any')
 )    
 
-# load conditions from the global tag
-process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
-from Configuration.AlCa.GlobalTag import GlobalTag
-process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_mc', '')
```

Comment:
In the unit-tests, the use of the GT is not strictly needed, so I thought it would be better to remove it.